### DISCUSSION TITLE
Little problem with two st7735 screens and animated eyes example.

### DISCUSSION THREAD

**Discussion by 2ant** [Bodmer/TFT_eSPI#2306] [Q&A - General] [answered]
Hi, my problem is that the first screen has both eyes on top of each other ( flickering), the second screen has only one eye as intended.

![20230108_131101](https://user-images.githubusercontent.com/52797081/211195391-c5a2999c-bcf2-44ca-8cca-f87fe704556b.jpg)

Each screen has a different CS pin defined : 15 for screen 1 and 0 for screen two.

I also tried this : 

```C
#include <TFT_eSPI.h> // Hardware-specific library
#include <SPI.h>

TFT_eSPI tft = TFT_eSPI();       // Invoke custom library


#define LargeScreenCS 15
#define SmallScreenCS 0

void setup(void) {
  pinMode(LargeScreenCS,OUTPUT);              // The CS pins will be outputs
  pinMode(SmallScreenCS,OUTPUT);
  // enable both screens for the general initialisation, applies to both
  digitalWrite(LargeScreenCS,0);
  digitalWrite(SmallScreenCS,0);
  tft.setRotation(3);
  tft.init();
  tft.fillScreen(0);
  
  digitalWrite(SmallScreenCS,1);        // switch off small screen, leaving just large one active
  tft.drawString("Large screen.",0,0);
  delay(2000);                          // wait a couple of secs before writing to small screen, just for dramatic effect!
  digitalWrite(SmallScreenCS,0);        // enable small screen
  digitalWrite(LargeScreenCS,1);   
  delay(500);     // tell large screen to ignore bus
  tft.drawString("Small screen.",0,0);  // write the text
}

void loop() {}
```

and after 2500ms i end up with "small screen" writtent on both screen.

From what i understand for screen one to receive data CS must be low but since it is specifficaly on high and the problem happens also on the animated eyes example the answer must be something else.

Any help is appreciated.

**Comment by 2ant** [comment, marked as answer]
Just tried with the pin 16 for screen one and everything works. 
I will mark as solved but i still don't know what was the problem. 

The board was an ESP32 Dev kit C V2 from AZ-Delivery

![image](https://user-images.githubusercontent.com/52797081/211196686-42ac0e99-4da0-4648-8b92-5966183cc584.png)

**Comment by patgau215** [comment]
I had the same problem, with one screen having two eyes. I found changing the User_Setup.h for the board I am using and commented out the TFT_CS pin line. I let the program "Animated_Eyes_1" define the TFT1_CS pin. This fixed that problem.  

the board is a Nucleo F401RE and TFT screen are 1.54" ST7789 240x240 from Adafriut.
